src/gasman.h

```diff
@@ -553,6 +553,18 @@ EXPORT_INLINE UInt ResizeWordSizedBag(Bag bag, UInt size)
 }
 
 
+#if defined(SYS_IS_CYGWIN32) && defined(SYS_IS_64_BIT)
+EXPORT_INLINE void ENSURE_BAG(Bag bag)
+{
+    memset(PTR_BAG(bag), 0, SIZE_BAG(bag));
+}
+#else
+EXPORT_INLINE void ENSURE_BAG(Bag bag)
+{
+}
+#endif
```

Comment:
> At the very least, there should be a comment here that explains what the purpose of ENSURE_BAG is; else this risks becoming a maintenance nightmare down the road.

Of course :)  As I said, I just wanted to avoid spending time on documentation until/unless the approach was deemed acceptable.  Of course, if there's anything I need to explain better in this PR I should do that.

> But I also don't quite understand why this is exposed globally -- I thought this is meant specifically to work around an issue with the GMP assembler code? Then I'd expect this function to be localized to src/integer.c. Or do you have any reason to expect that it might be needed in additional files in the near future?

A fair question--I debated this myself.  Currently I do not anticipate any other use.  It's conceivably useful in any GAP packages that happen to interface with libraries that also use han-written assembly routines and thus has the same possible problem.  I am not so familiar with the universe of GAP packages though so it's hard for me to know.

Until in the remote chance that another use for this is discovered I'm happy to just localize this workaround to integer.c.

> Finally, please rewrite it like this, to reduce code duplication:

Sure; will do.

Reply:
Sorry, missed the bit about waiting with documentation etc., my bad (and of course that's a very reasonable approach to avoid wasted time).

Please make `ENSURE_BAG` local (we can "publish" it if anybody determines a need for it, though I'd probably recommend to package authors to simply copy it instead). I promise to approve this once comments etc. are in there. :-)